src/scss/_Billboard.scss

```diff
@@ -0,0 +1,16 @@
+@import 'variables';
+
+.MovieDetails-Billboard {
```

Comment:
I agree that we need to display the Ratings and Details info here on the Billboard. Make some transparent white backgrounds for the text to live on and give the backdrop image some high opacity! I think it will look rad.

Reply:
Totally with ya there! that sounds like a good plan